bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/OpenUVIconProvider.java

```diff
@@ -115,10 +115,12 @@ private String getResource(String iconName) {
         String result = "";
 
         URL iconResource = context.getBundle().getEntry("icon/%s.svg".formatted(iconName));
-        try (InputStream stream = iconResource.openStream()) {
-            result = new String(stream.readAllBytes(), StandardCharsets.UTF_8);
-        } catch (IOException e) {
-            logger.warn("Unable to load ressource '{}' : {}", iconResource.getPath(), e.getMessage());
+        if (iconResource != null) {
```

Comment:
Should anything be logged if we didn't find an entry here (like we log when unable to load the resource)? For example, the `iconName` might be relevant to know?

Reply:
You're more than correct. Let me correct this.